types/allow.go

```diff
@@ -0,0 +1,14 @@
+// Copyright 2022 Juan Pablo Tosso and the OWASP Coraza contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package types
+
+// AllowType represents the values that the allow disruptive action can take
+type AllowType int
```

Comment:
Looks like this isn't exposed in a public API so we should keep the enum internal too

Reply:
You are right!

Reply:
I'm gonna piggyback on @jptosso changes later today.